Assets/Scripts/ThrowSword.cs

```diff
@@ -49,7 +49,7 @@ void PlayerThrow()
     IEnumerator EnemyThrow()
     {
         yield return new WaitForSeconds(3);
-        Instantiate(swordPrefab, swordThrowPoint.position, Quaternion.identity);
+        Instantiate(swordPrefab, swordThrowPoint.position, swordThrowPoint.rotation);
```

Comment:
you can try the knight rotation instead for the sword, it might just work to flipping the sword for you